docs/control-plane-guide/api.mdx

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: API
+title: Sending a command via the API
+description: Sending a control plane command via the API
+---
+
+# Sending a command via the REST API
+
+## Required information
+
+There are three pieces of information that are required to send a command to an Edge Authorizer via the REST API:
+1. The Edge Authorizer's instance ID
+2. The Tenant ID for the account/organization
+3. The Discovery API key for the tenant
+
+:::tip Tip
+See the script [below](#scripting-commands) for a way to obtain the Instance ID for an Aserto Tenant and Edge Authorizer connection, and send a command to that instance.
+:::
+
+### Instance ID
+
+The instance ID shows up in the Authorizers tab:
+
+<img src="/control-plane-instance-id.png" alt="control plane instance ID" />
+
+### Tenant ID
+
+For a personal account, the tenant ID can be obtained from the Account Settings page. For an organization, the tenant ID can be obtained from the Manage Organization page.
+
+### Discovery API key
+
+The Discovery API key can be obtained from the Connections tab:
+
+<img src="/discovery-api-key.png" alt="discovery api key" />
+
+## Available commands
+
+Two commands are available to send to connected Edge Authorizers:
+* Sync Policy Instance: this will short-circuit the OPA Discovery timer and force an immediate download of a policy image
```

Comment:
It's a pet peeve of mine and feel free to ignore, but documentation is typically written in the present tense, not future. It describes what things _are_ and what they _do_.
I'd phrase this as:
```
* Sync Policy Instance: this short-circuits the OPA Discovery timer and forces an immediate download of a policy image.
* Sync Edge Directory: this short-circuits the Directory timer and forces an immediate sync of any new manifest or data in the Central Directory